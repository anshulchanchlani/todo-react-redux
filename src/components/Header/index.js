import React, { Component } from 'react';
import './index.scss';
class Header extends Component {
    constructor(props) {
        super(props);
        this.state = {isLoggedIn:true  };
    }
    render() {
        return (
            this.state.isLoggedIn && 
            <header className="app-header">
                <h1>Todo's</h1>
            </header>
        );
    }
}

export default Header;