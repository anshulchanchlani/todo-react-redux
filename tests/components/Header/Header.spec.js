import React from 'react';
import Header from '../../../src/components/Header';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import {shallow} from 'enzyme';


describe('Header tests',()=>{
    const wrapper = shallow(<Header/>); 
    
    test('Header renders',()=>{
      expect(toJson(wrapper)).toMatchSnapshot();
    })

    test('Header has a heading',()=>{
        expect(wrapper.find('h1').length).toEqual(1)
    })
    
})
