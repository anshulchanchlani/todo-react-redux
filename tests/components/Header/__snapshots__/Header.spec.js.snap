// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests Header renders 1`] = `
<header
  className="app-header"
>
  <h1>
    Todo's
  </h1>
</header>
`;
